docker/kubernetes-agent-tentacle/Dockerfile

```diff
@@ -49,8 +49,9 @@ RUN chgrp -R 0 /opt /usr /.dotnet && \
 
 # The same as above, but we shouldn't recursively own /etc, as it would add
 # ownership to system files like /etc/passwd
-RUN chgrp 0 /etc && \
-    chmod g=u /etc
+# We own /etc/ssl/certs so we can do a rehash
+RUN chgrp 0 /etc /etc/ssl/certs && \
+    chmod g=u /etc /etc/ssl/certs
```

Comment:
When running as a non-root user, we don't have permission to do the ssl cert rehash at the start. This allows for it